src/Randomizer.Shared/Models/Seed.cs

```diff
@@ -1,19 +1,21 @@
 ﻿using System.Collections.Generic;
+using System;
+using System.ComponentModel.DataAnnotations.Schema;
 
-namespace Randomizer.Shared.Models {
-
-    public class Seed {
+namespace Randomizer.Shared.Models
+{
+    public class Seed
```

Comment:
I know you just repurposed this file but I think we can come up with a better name than "seed" because "seed" to me is the number used for the RNG, not the entire generated game. Maybe not an entirely different name, but at least enough to clearly differentiate between the generated game "seed" and the RNG seed

Reply:
For our purposes, probably something like "GeneratedRom" or something like that would be more accurate?

Reply:
Yeah, I think that would work